src/ActionModal/components/SendModal.vue

```diff
@@ -205,7 +205,11 @@ export default {
         toAddress: this.address,
         amounts: [
           {
-            amount: uatoms(+this.amount),
+            amount: toMicroUnit(
+              +this.amount,
```

Comment:
is this `+` on purpose?

Reply:
would convert to a number. don't know if that is needed. wil lcheck

Reply:
removed